mmseg/datasets/pipelines/transforms.py

```diff
@@ -5,6 +5,10 @@
 import numpy as np
 from mmcv.utils import deprecated_api_warning, is_tuple_of
 from numpy import random
+import cv2
+import albumentations
+from albumentations import Compose
```

Comment:
As `albumentations` is not tightly coupled with mmseg, I think it would be better that mmseg can be installed without `albumentations`
```suggestion
try:
    import albumentations
    from albumentations import Compose
except ImportError:
    albumentations = None
    Compose = None
```